server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -28,17 +28,17 @@ Feature: GetMessages method
     And the list of messages is empty
 
   Scenario: Retrieving message should return not found when message doesn't exist
-    When the user ask for messages "["username@domain.tld|inbox|12"]"
+    When the user ask for an unknown message
     Then no error is returned
-    And the notFound list should contains "username@domain.tld|inbox|12"
+    And the notFound list should contain the requested message id
 
   Scenario: Retrieving message should return messages when exists
-    Given the user has a message in "inbox" mailbox with subject "my test subject" and content "testmail"
-    When the user ask for messages "["username@domain.tld|inbox|1"]"
+    Given the user has a message "m1" in "inbox" mailbox with subject "my test subject", content "testmail"
+    When the user ask for messages "m1"
     Then no error is returned
     And the list should contain 1 message
-    And the id of the message is "username@domain.tld|inbox|1"
-    And the threadId of the message is "username@domain.tld|inbox|1"
+    And the id of the message is "m1"
```

Comment:
We can't guess the id of the message, so we just give them a name during the feature.
Then in the stepdef we are keeping them in a Map<message name, message id>.